### PR TITLE
Retrieve value of editor.parameterHints.enabled

### PR DESCRIPTION
**Pull request by Bernardin-MS**
This addresses issue #9314

### CODE REVIEW

**Review by Colengms — APPROVED** [review]

File: Extension/src/LanguageServer/settings.ts [hunk 1]
```diff
@@ -959,6 +960,7 @@ export class OtherSettings {
     public get filesExclude(): vscode.WorkspaceConfiguration | undefined { return vscode.workspace.getConfiguration("files", this.resource).get("exclude"); }
     public get filesAutoSaveAfterDelay(): boolean { return vscode.workspace.getConfiguration("files").get("autoSave") === "afterDelay"; }
     public get InlayHintsEnabled(): boolean { return vscode.workspace.getConfiguration("editor.inlayHints").get<string>("enabled") !== "off"; }
+    public get InlineParameterHintsEnabled(): boolean | undefined { return vscode.workspace.getConfiguration("editor.parameterHints").get<boolean>("enabled"); }
```
Comment (michelleangela) [inline Extension/src/LanguageServer/settings.ts]: Change "InlineParameterHintsEnabled" to "ParameterHintsEnabled" and other associated references.

**Review by michelleangela — CHANGES_REQUESTED** [review]

File: Extension/src/LanguageServer/settings.ts (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/src/LanguageServer/settings.ts]: Should these all be camelCase?

File: Extension/src/LanguageServer/settings.ts [hunk 2]
```diff
@@ -959,6 +960,7 @@ export class OtherSettings {
     public get filesExclude(): vscode.WorkspaceConfiguration | undefined { return vscode.workspace.getConfiguration("files", this.resource).get("exclude"); }
     public get filesAutoSaveAfterDelay(): boolean { return vscode.workspace.getConfiguration("files").get("autoSave") === "afterDelay"; }
     public get InlayHintsEnabled(): boolean { return vscode.workspace.getConfiguration("editor.inlayHints").get<string>("enabled") !== "off"; }
```
Comment (sean-mcmanus) [inline Extension/src/LanguageServer/settings.ts]: I think these should be named editorInlayHintsEnabled (pre-existing issue) and editorParameterHintsEnabled to be consistent with the naming of the other settings. Looks like settingsEditor could be changed to workbenchSettingsEditor too (pre-existing issue). I think these methods should be alphabetically sorted too. At a minimum it should be camelCased.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/src/LanguageServer/settings.ts (hunk 1 above)
Comment (michelleangela) [reply Extension/src/LanguageServer/settings.ts]: Oh yes, they should be. I asked Bernardin to copy pattern of `InlayHintsEnabled()`, but that wasn't in camelCase by mistake. 

Bernardin, could you fix `InlayHintsEnabled()` to camelCase to if you're going to change `ParameterHintsEnabled()` to camelCase.

File: Extension/src/LanguageServer/settings.ts (hunk 2 above)
Comment (Bernardin-MS) [reply Extension/src/LanguageServer/settings.ts]: Are you talking about these lines? https://github.com/microsoft/vscode-cpptools/blob/8a004672460fc23ea5fa0724148af43c60fab661/Extension/src/LanguageServer/settings.ts#L962-L963
https://github.com/microsoft/vscode-cpptools/blob/8a004672460fc23ea5fa0724148af43c60fab661/Extension/src/LanguageServer/settings.ts#L965

Comment (sean-mcmanus) [reply Extension/src/LanguageServer/settings.ts]: Yes

**Review by sean-mcmanus — APPROVED** [review]

**Review by michelleangela — APPROVED** [review]

**Pull request merged**